meteor/client/ui/RundownView/RundownTiming.tsx

```diff
@@ -332,7 +332,7 @@ withTracker<IRundownTimingProviderProps, IRundownTimingProviderState, IRundownTi
 					this.displayDurationGroups[partInstance.part.displayDurationGroup] =
 						(this.displayDurationGroups[partInstance.part.displayDurationGroup] || 0) + (partInstance.part.expectedDuration || 0)
 					displayDurationFromGroup = partInstance.part.displayDuration
-						|| Math.max(0, this.displayDurationGroups[partInstance.part.displayDurationGroup], this.props.defaultDuration || DEFAULT_DURATION)
+						|| Math.max(0, this.displayDurationGroups[partInstance.part.displayDurationGroup], partInstance.part.gap ? 1 : this.props.defaultDuration || DEFAULT_DURATION)
```

Comment:
This allows the gap to fold down to being essentially invisible (a value of 0 did not play nicely at all). So, the user can see their time running out entirely - without this check it would only compress down to the default duration and no further.

Reply:
Aha, OK, that makes sense. OK, in that case I'm fine with getting this to a constant like `MINIMAL_NONZERO_DURATION` and perhaps a comment on why this is the way it is.

Reply:
Addressed by https://github.com/olzzon/tv-automation-server-core/commit/f0efaa08f1009f538b409ccccebb64beaf8894a2